lms/static/sass/discussion/elements/_editor.scss

```diff
@@ -87,13 +87,18 @@
   .wmd-button {
     position: absolute;
     display: inline-block;
-    padding-right: 3px;
-    padding-left: 2px;
     width: 20px;
     height: 20px;
     background: none;
     list-style: none;
     cursor: pointer;
+    border: none;
```

Comment:
Are we still concerned with property order? If so, `border` should come after `height`, but before `padding`.